src/db/migrations/20231102024624_add_unique_index_to_scan_results_id.js

```diff
@@ -24,6 +24,8 @@ export async function up(knex) {
  */
 export function down(knex) {
   return knex.schema.table("onerep_scan_results", (table) => {
+    // @ts-ignore TODO: Determine if the following line should be changed to
+    // `defaultTo` or remain unchanged as initially deployed.
```

Comment:
Same as above re: `dropUnique`

Reply:
Ah, yes. I’ve copied the wrong `ts-ignore` comment over. The line is still getting highlighted by the linter complaining that it should be `table.dropUnique(["onerep_scan_id"]);` it also just takes a `string`.